Python/Product/Uwp/Templates/Projects/BackgroundService/PythonBackgroundService.pyproj

```diff
@@ -55,4 +55,9 @@
 
   <Import Project="$(PtvsTargetsFile)" />
 
+  <PropertyGroup>
+    <TargetPlatformVersion>$winsdkversion$</TargetPlatformVersion>
+    <TargetPlatformMinVersion>$winsdkminversion$</TargetPlatformMinVersion>
+  </PropertyGroup>
+
```

Comment:
Do you really want these at the end and not before the targets file is imported? What if you later need to add conditions based on the target version?